cardano-api/src/Cardano/Api/Orphans.hs

```diff
@@ -24,3 +30,15 @@ deriving instance NFData (VerKeyDSIGN (DSIGN c)) => NFData (DiscVKey kd c)
 
 deriving instance Generic (SKey c)
 deriving instance NFData (SignKeyDSIGN (DSIGN c)) => NFData (SKey c)
+
+deriving instance NoUnexpectedThunks (SKeyES TPraosStandardCrypto)
+deriving instance Generic (SKeyES c)
+deriving instance NFData (SignKeyKES (KES c)) => NFData (SKeyES c)
+
+deriving instance Generic Ed448DSIGN
+deriving instance NFData (SignKeyKES (SimpleKES Ed448DSIGN))
```

Comment:
I think we can now depend on the real one, Ed25519, rather than the place-holder Ed448.

Reply:
@dcoutts I don't think we can as yet: https://github.com/input-output-hk/ouroboros-network/blob/918b8d4ca2df20abfe8bbaf21fe0eb86d5b9a2fe/ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Protocol/Crypto.hs#L53

Reply:
Right, we need to wait for it to be updated in `ouroboros-consensus-shelley`.

Reply:
https://github.com/input-output-hk/ouroboros-network/pull/1984